bbb_view.php

```diff
@@ -129,6 +129,17 @@
         bigbluebuttonbn_event_log(\mod_bigbluebuttonbn\event\events::$events['meeting_left'], $bigbluebuttonbn);
         // Update the cache.
         $meetinginfo = bigbluebuttonbn_get_meeting_info($bbbsession['meetingid'], BIGBLUEBUTTONBN_UPDATE_CACHE);
+        // Check the origin page.
+        $sql = "SELECT meta FROM {bigbluebuttonbn_logs}
+                  where userid = ? AND (log = 'Join' OR log = 'Create')
+                  ORDER BY ID DESC LIMIT 1";
+        $params = array('userid' => $SESSION->bigbluebuttonbn_bbbsession['userID']);
```

Comment:
I know this is a horrible implementation but you have already $bbbsession defined. No need to use the global. Just to be consistent with the rest of the code.